### ISSUE TITLE
Module not found; using Python 3.11.2.

### ISSUE THREAD

**Issue by dpenfield9** [dannytsang/pyubibot#2]
Hello Danny,
I have installed pyubibot and can see it in the site-packages directory.
At "from pyubibot import Channel" I get "No module named pyubibot".
This is on a Raspberry Pi running Raspbian.
As a test I also tried it on Python 3.11.2 on Windows 10. I get the same result.
I have installed other modules for the same project e.g. python-kasa and have no issues with these.
Would you advise, please?
Kind regards,
David.

#!/usr/bin/env python3
#The line above specifies the interpreter to be used
#Import the required libraries
import sys
print("Using Python ", end="", flush = True)
print(sys.version)
print(sys.path)
import json
import os
from pyubibot import Channel

**Comment by dannytsang** [comment]
I will take a look and get back to you.

**Comment by dannytsang** [comment]
Sorry for the delay @dpenfield9 

To use the package you need to import the `Ubibot` class. Here's an example:

```python
# import class Ubibot from pyUbibot package.
from pyubibot import Ubibot
# Instantiate the the Ubibot with your API key.
u = Ubibot('MY_API_KEY')
# Example getting data from Ubibot by getting channel information.
u.get_channels()
```

**Comment by dpenfield9** [comment]
Hi Danny,
Thank you for the response.

For the line:
from pyubibot import Ubibot
I still get:
ModuleNotFoundError: No module named 'pyubibot'

In case it helps, I used "pip install pyubibot" to install.
python -V gives "Python 3.11.2".

Please advise.

Kind regards,
David.

**Comment by dannytsang** [comment]
It was my fault. The module didn't include any of the Python code. This should be fixed in 0.0.4. If you run `pip install --upgrade pyubibot` it should upgrade and then the module should import correctly.

**Comment by dannytsang** [comment]
Odd that it wasn't installed as part of it. I think I need to work on the dependency list but glad you figured it out.

I will work on some better documentation. For the time being, you should be able to use the [above example](https://github.com/dannytsang/pyubibot/issues/2#issuecomment-1528981759) to get started and inspect the output to drill down the what you may need.
